yeat/tests/test_cli.py

```diff
@@ -40,11 +41,6 @@ def test_basic_dry_run(tmp_path):
     cli.main(args)
 
 
-def test_no_args():
-    with pytest.raises(SystemExit, match=r"2"):
-        cli.main(None)
```

Comment:
Removed this test overall. Coverage went down slightly.

This test works fines when ran by itself, but fails when running the test suite.

In order for this test to pass, you'll need to do this:
>def test_no_args():
     with pytest.raises(argparse.ArgumentError, match=r"yeat is not an integer"):
         cli.main(None)

Again, this problem touches the `type=check_positive`.